sign/options.go

```diff
@@ -16,13 +16,37 @@ limitations under the License.
 
 package sign
 
+const (
+	DefaultFulcioURL     = "https://v1.fulcio.sigstore.dev"
+	DefaultRekorURL      = "https://rekor.sigstore.dev"
+	DefaultOIDCIssuerURL = "https://oauth2.sigstore.dev/auth"
+)
+
 // Options can be used to modify the behavior of the signer.
 type Options struct {
 	// Verbose can be used to enable a higher log verbosity
 	Verbose bool
+	// Experimental enable the cosing experimental feature. Enabled by default
+	Experimental          bool
+	OutputSignaturePath   string
+	OutputCertificatePath string
+	Annotations           map[string]interface{}
+	KeyPath               string
+	FulcioURL             string
+	RekorURL              string
+	OIDCIssuer            string
+	UploadToTLOG          bool
+	Force                 bool
```

Comment:
I would not expose so many configuration options in the first iteration, only those we really require for now.

Reply:
would hardcode the ones we need to set?

Reply:
Yes, exactly. Not exposing too much API surface for now so we can iterate on what we want.

Reply:
i would like to keep the keypath because then i can use locally for testing and don't need oidc :)